docs/section10/10-14.md

```diff
@@ -0,0 +1,167 @@
+
+**Rule Assertion:** B-RMR = expected value                                           
+**Appendix G Section:** Section G3.1.2.1 Equipment Efficiencies  
+**Appendix G Section Reference:** None  
+**Data Lookup:** Tables G3.5.2, G3.5.4 and G3.5.5  
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem   
+
+**Applicability Checks:**  
+
+1. Applicable to baseline HVAC system types 2, 3, 3a, 4, and 9.  
+
+**Function Call:**  
+
+1. get_baseline_system_types()  
+2. get_hvac_zone_list_w_area()  
+3. baseline_system_type_compare()  
+
+## Rule Logic:   
+**Applicability Check 1:**  
+- Get baseline hvac system type dictionary: `baseline_hvac_sys_type_ids_dict_b = get_baseline_system_types(B_RMR)`
+- Create a list of the target system types: `target_system_types = [HVAC_SYS.SYS_2, HVAC_SYS.SYS_3, HVAC_SYS.SYS_3a, HVAC_SYS.SYS_4, HVAC_SYS.SYS_9]`   
+- For each hvac system in the B_RMR: `for hvac_b in B_RMR...HeatingVentilationAirConditioningSystem:`  
+    Check if the hvac system is applicable to this check based on system type  
+    - Get the hvac system type for hvac_b: `hvac_sys_type_b = baseline_hvac_sys_type_ids_dict_b.keys()[list(baseline_hvac_sys_type_ids_dict_b.values()).index(hvac_b.id)]`
+    - Check if system type is of an applicable system type, if yes then carry on, if no then NOT_APPLICABLE for the hvac system:  `if any(baseline_system_type_compare(hvac_sys_type_b, target_sys_type, true) for target_sys_type in target_system_types):`  
+        - Reset the efficiency_matches_acceptable_value boolean variable to FALSE: `efficiency_matches_acceptable_value = FALSE`  
+        - Reset the cooling capacity not defined variable to false: `was_cool_capacity_not_defined = FALSE`
+        - Reset the heating capacity not defined variable to false: `was_heat_capacity_not_defined = FALSE`
+        Get the attributes of the hvac system relevant to this check   
+        - Assign the heating system to an object (there should only be 1 heating system): `heating_system_b = hvac_b.heating_system` 
+        - Assign the cooling system to an object (there should only be 1 cooling system): `cooling_system_b = hvac_b.cooling_system` 
+
+        Determine the expected modeled efficiency based on the system type and capacity and compare to modeled value:
+        - SYSTEM TYPE 2  
+        - Check if the system is system type 2: `if hvac_sys_type_b = "SYS-2":`
+            - Loop through each of the efficiency_metric_types associated with the heating system until FULL_LOAD_EFFICIENCY_NO_FAN is found. Get the associated value of the efficiency from the list of efficiency_metric_values associated with the heating_system_b : `For x in range(len(heating_system_b.efficiency_metric_types)):`
```

Comment:
`FULL_LOAD_EFFICIENCY_NO_FAN` should be `FULL_LOAD_COEFFICIENT_OF_PERFORMANCE_NO_FAN`

Reply:
Change to HEAT_PUMP_COEFFICIENT_OF_PERFORMANCE_HIGH_TEMPERATURE_NO_FAN since in AHRI 310 the rating condition is 47F